pkg/config/planargument.go

```diff
@@ -0,0 +1,31 @@
+package config
+
+import (
+	"github.com/lunarway/shuttle/pkg/git"
+	"os"
+	"path"
+	"strings"
+)
+
+func isPlanArgumentAFilePlan(planArgument string) bool {
+	return strings.Index(planArgument, "/") == 0 || strings.Index(planArgument, "./") == 0 || strings.Index(planArgument, "../") == 0
```

Comment:
Could be implemented with [`strings.HasPrefix()`](https://golang.org/pkg/strings/#HasPrefix) for readability.

```suggestion
	return strings.HasPrefix(planArgument, "/") || strings.HasPrefix(planArgument, "./") || strings.HasPrefix(planArgument, "../")
```